apps/zero-runtime-next-app/src/components/Grid.tsx

```diff
@@ -1,6 +1,16 @@
 import { styled } from '@mui/zero-runtime';
+import type { Theme } from '@mui/material';
 
-const Grid = styled('div')(({ theme }: any) => ({
+declare module '@mui/zero-runtime' {
```

Comment:
This should be move to a different file, it shouldn't live in component's life. I am mentioning this because people that see this project may use this as a reference for how to define their custom theme.

Reply:
Done